Platforms/QemuQ35Pkg/QemuQ35Pkg.dsc

```diff
@@ -840,10 +844,8 @@
     gEfiMdeModulePkgTokenSpaceGuid.PcdDeviceStateBitmask|0x20
   !endif
 
-  # Set to debug if debugger is enabled.
-  !if $(DEBUGGER_ENABLED) == TRUE
-    gEfiMdeModulePkgTokenSpaceGuid.PcdDeviceStateBitmask|0x08
-  !endif
+  # Set to debug as debugger is enabled.
+  gEfiMdeModulePkgTokenSpaceGuid.PcdDeviceStateBitmask|0x08

```

Comment:
This will cause the default 30 second timeout. Do we want to drop that down to like 5 seconds or something to not slow down pipelines, test, etc?

Reply:
Does it make sense to add a build flag to not set this that can be used in cases like pipelines?

Reply:
@cfernald how will this cause the 30 second timeout? I thought this just controls whether the debugger can be enabled or not: https://github.com/microsoft/mu_feature_debugger/blob/24f5ebe0782b35e13e1529c405f94f9d776feae9/DebuggerFeaturePkg/DebugConfigPei/DebugConfigPei.c#L42.

Reply:
Yes, but sense you do not specify the PCD's you will use the defaults: https://github.com/cfernald/mu_feature_debugger/blob/62f972757843a1c9b583efd681ddac6f65ac11bf/DebuggerFeaturePkg/DebuggerFeaturePkg.dec#L52

Reply:
@cfernald , but I am specifying the `PcdDebugConfigFlags` saying no initial breakpoint, so it doesn't matter what the default is. So with the other comment, I will change this to specifying timeout = 0 always, but the initial breakpoint is off, unless you set the force flag. Then CI is not affected as by default no initial breakpoint, if you are overriding from the commandline, that is you saying yes, I want to break in, so wait for me forever. If the cli override is not set, you will not have an initial break in and will only break in if an exception occurs.

Reply:
Ah, sorry i missed that line in my original pass. you are right.